doc/howto/cluster_recover.md

```diff
@@ -6,43 +6,38 @@ In that case, no operations are possible on this member, and neither are operati
 See {ref}`clustering-offline-members` and {ref}`cluster-automatic-evacuation` for more information.
 
 If you can bring the offline cluster members back or delete them from the cluster, operation resumes as normal.
-If this is not possible, there are a few ways to recover the cluster, depending on the scenario that caused the failure.
-See the following sections for details.
+If the cluster members are lost permanently (e.g. disk failure), it is possible to recover them if at least one member survives.
 
 ```{note}
-When your cluster is in a state that needs recovery, most `lxc` commands do not work, because the LXD client cannot connect to the LXD daemon.
+When your cluster is in a state that needs recovery, most `lxc` commands do not work, because the LXD daemon cannot connect to the LXD database.
 
-Therefore, the commands to recover the cluster are provided directly by the LXD daemon (`lxd`).
+The commands to recover a cluster are provided directly by the LXD daemon (`lxd`).
 Run `lxd cluster --help` for an overview of all available commands.
 ```
 
-## Recover from quorum loss
+## Database members
 
 Every LXD cluster has a specific number of members (configured through {config:option}`server-cluster:cluster.max_voters`) that serve as voting members of the distributed database.
-If you permanently lose a majority of these cluster members (for example, you have a three-member cluster and you lose two members), the cluster loses quorum and becomes unavailable.
-However, if at least one database member survives, it is possible to recover the cluster.
+If you lose a majority of these cluster members (for example, you have a three-member cluster and you lose two members), the cluster loses quorum and becomes unavailable.
+
+To determine which members have (or had) database roles, log on to any surviving member of your cluster and run the following command:
 
-To do so, complete the following steps:
+    sudo lxd cluster list-database
 
-1. Log on to any surviving member of your cluster and run the following command:
+## Recover from quorum loss
 
-       sudo lxd cluster list-database
+If only one cluster member with the database role survives, complete the following steps:
```

Comment:
Do you really mean "only one"? So if I have a 5-member cluster and two members with database role are left, I can't recover?

Reply:
Yes... `recover-from-quorum-loss` turns the current member into the only member of the Dqlite cluster. `cluster edit` should be used if more than one member needs to be recovered.

I think it's worth discussing if `recover-from-quorum-loss` is even needed; its outcome can be functionally achieved with `cluster edit` (even if the end state is subtly different). cc @tomponline

Reply:
Ah okay - I think we should (at least) point to that other section then from here. Because my first question as a reader was "what if not?".